Marlin/Marlin_main.cpp

```diff
@@ -1603,6 +1603,7 @@ inline void set_homing_bump_feedrate(AxisEnum axis) {
     SERIAL_ECHOLNPGM("Warning: Homing Bump Divisor < 1");
   }
   feedrate_mm_m = homing_feedrate_mm_m[axis] / hbd;
+  return feedrate_mm_m;
```

Comment:
When my `run_z_probe()` changes (https://github.com/AnHardt/Marlin/pull/57) are in `set_homing_bump_feedrate()` can go to `get_homing_bump_feedrate(AxisEnum axis)` and
```
feedrate_mm_m = homing_feedrate_mm_m[axis] / hbd;
```
can go.

I'm still not sure how to name the probing speeds. (https://github.com/AnHardt/Marlin/pull/57#discussion_r70615384)